tests/keras_autolog/test_keras_autolog.py

```diff
@@ -129,15 +150,15 @@ def test_keras_autolog_logs_expected_data(keras_random_data_run):
     assert data.params['optimizer_name'] == 'Adam'
     assert 'epsilon' in data.params
     assert data.params['epsilon'] == '1e-07'
-    assert 'model_summary' in data.tags
-    assert 'Total params: 6,922' in data.tags['model_summary']
-    artifacts = client.list_artifacts(keras_random_data_run.info.run_id)
-    artifacts = map(lambda x: x.path, artifacts)
-    assert 'model_summary.txt' in artifacts
```

Comment:
Since we want to log the model summary as an artifact regardless of text length, we should add this line back in.